index.js

```diff
@@ -78,10 +85,15 @@ exports.init = function (sbot, config) {
     )
   }
 
-  sbot.getSubset = function getSubset(queryObj, opts) {
+  sbot.getSubset = function getSubset(query, opts) {
     if (!opts) opts = {}
+    const querylang = opts.querylang || 'ssb-ql-0'
```

Comment:
This makes `getSubset` default to `ssb-ql-0`. I'm not sure which one should be the default. Thoughts @arj03 ?

Reply:
I think that should be an error. We don't want to assume the format of the query if it is not specified.